apps/loan/src/components/PageLlamaMarkets/LendTableFooter.tsx

```diff
@@ -0,0 +1,56 @@
+import Card from '@mui/material/Card'
+import Link from '@mui/material/Link'
+import ArrowOutwardIcon from '@mui/icons-material/ArrowOutward'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Grid from '@mui/material/Grid2'
+import Typography from '@mui/material/Typography'
+import type { ReactNode } from 'react'
+import { t } from '@lingui/macro'
+import { SoftLiquidationIcon } from '@/components/PageLlamaMarkets/SoftLiquidationIcon'
+import { CardStackPlusIcon } from '@/components/PageLlamaMarkets/CardStackPlusIcon'
+import { SignIcon } from '@/components/PageLlamaMarkets/SignIcon'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+const GridItem = ({ children, title, icon }: { children: ReactNode; title: string; icon: ReactNode }) => (
+  <Grid size={{ tablet: 4, mobile: 12 }}>
+    {icon}
+    <Typography variant="headingXsBold">{title}</Typography>
+    <Typography variant="bodyMRegular">{children}</Typography>
+  </Grid>
+)
+
+export const LendTableFooter = () => (
+  <Card sx={{ paddingX: Spacing.lg, paddingTop: Spacing.lg, backgroundColor: (t) => t.design.Layer[1].Fill }}>
+    <Typography variant="headingSBold" textAlign="center">{t`Why Use Llamalend?`}</Typography>
+    <Box paddingY={Spacing.md}>
```

Comment:
As suggested above you might get rid of this Box, but if you don't, not sure where the `Spacing.md` is in the Figma design, I can't find it anywhere.

Reply:
It's here. I replaced with rowGap as you suggested
![image](https://github.com/user-attachments/assets/f23cb6ac-01f5-4b75-adda-ead696fa8c07)